action.sh

```diff
@@ -203,6 +203,8 @@ function start_vm {
 	cat <<-EOF > /etc/systemd/system/shutdown.sh
 	#!/bin/sh
 	sleep ${shutdown_timeout}
+	# ensure the active gcloud account in the shutdown script is the same one configured on instance creation
+	gcloud config set account \$(gcloud auth list --filter=status:ACTIVE --format=\"value(account)\")
```

Comment:
From testing, common usage of `gcloud auth` in actions doesn't affect the active gcloud project so I haven't added a `gcloud config set project` command here.